main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"github.com/kinvolk/mountinfo-graph/bindata"
+	"github.com/kinvolk/mountinfo-graph/migraph"
+	"html/template"
+	"net/http"
+)
+
+func indexHandler(w http.ResponseWriter, r *http.Request) {
+	t, err := template.New("index").Parse(bindata.Index())
```

Comment:
You could probably put the contents of this function into `func generateFromTemplate(htmlTemplate string, body string)` and then use the new function in both `indexHandler` and `showHandler`.